packages/foam-cli/src/commands/janitor.ts

```diff
@@ -0,0 +1,78 @@
+import { Command, flags } from '@oclif/command';
+import * as ora from 'ora';
+import { initializeNoteGraph, generateLinkReferences, generateHeading, getKebabCaseFileName } from 'foam-core';
+import * as path from 'path';
+import { applyTextEdit } from '../utils/apply-text-edit';
+import { writeFileToDisk } from '../utils/write-file-to-disk';
+import { renameFile } from '../utils/rename-file';
+
+
+export default class Janitor extends Command {
+  static description = 'Updates link references and heading across all the markdown files in the given workspaces';
+
+  static examples = [
+    `$ foam-cli janitor path-to-foam-workspace
+Successfully generated link references and heading!
+`,
+  ]
+
+  static flags = {
+    help: flags.help({ char: 'h' }),
+  }
+
+  static args = [{ name: 'workspacePath' }]
+
+  async run() {
+    const spinner = ora('Reading Files').start();
+
+    const { args, flags } = this.parse(Janitor)
+
+    const { workspacePath = './' } = args;
+
+    const foamWorkspaceDir = path.join(__dirname, workspacePath);
```

Comment:
I don't think we need to join here? Just `workspacePath` will do?

Reply:
Let's also check that the directory exists, and exit if it doesn't.

Reply:
Yeah @jevakallio workspacePath works, changing it.